secure_message_v2/application.py

```diff
@@ -24,14 +33,30 @@ def create_app():
     app.register_blueprint(messages_bp, url_prefix="/messages")
     app.register_blueprint(threads_bp, url_prefix="/threads")
 
+    @app.errorhandler(UAAError)
+    @app.errorhandler(JWTValidationError)
+    def handle_exception(e):
+        logger.error(e.message)
+        response = jsonify({"error": "Unauthorized"})
+        response.status_code = 401
+        return response
+
+    @app.before_request
```

Comment:
Appreciate this is a consequence of us moving the scope of this PR but given what we have discussed, and now we know UAA isn't involved in the token verification / authentication, we shouldn't be making this request before every request. I do think there's value in leaving this implementation for the next story so maybe we just remove the `before_request` for the time being.

On a positive note it does demonstrate that this client request to UAA is working 👍 

```
uaa | ClientAuthenticationSuccess ('Client authentication success'): principal=secure_message, origin=[remoteAddress=172.19.0.24, clientId=secure_message], identityZoneId=[uaa]
uaa | TokenIssuedEvent ('["scim.userids","scim.me","scim.read"]'): principal=secure_message, origin=[caller=secure_message, details=(remoteAddress=172.19.0.24, clientId=secure_message)], identityZoneId=[uaa]
secure-message-v2 | "GET /info HTTP/1.1" 308 241 "-" "PostmanRuntime/7.37.3"
uaa | ClientAuthenticationSuccess ('Client authentication success'): principal=secure_message, origin=[remoteAddress=172.19.0.24, clientId=secure_message], identityZoneId=[uaa]
uaa | TokenIssuedEvent ('["scim.userids","scim.me","scim.read"]'): principal=secure_message, origin=[caller=secure_message, details=(remoteAddress=172.19.0.24, clientId=secure_message)], identityZoneId=[uaa]
secure-message-v2 | "GET /info/ HTTP/1.1" 200 36 "http://localhost:5051/info" "PostmanRuntime/7.37.3"
```